### ISSUE TITLE
bug(iOS): "Installed Languages" title of the page for keyboard search screen 

### ISSUE THREAD

**Issue by MakaraSok** [keymanapp/keyman#6519]
**Describe the bug**

<img width="497" alt="image" src="https://user-images.githubusercontent.com/28331388/162690119-c3e669e6-d6d2-4f82-a4dc-9d906f705717.png">

The title of the screen here is a little misleading if we look into it closer. It's a screen where the user looks for a keyboard to download and install on the phone. Now the title says "Installed Keyboard" which is a little misleading and not indicating the intend of what should be done on the screen.

**Expected behavior**

The title of the screen should be something a long this line, "Keyboard search" or "Download and install a keyboard".

---

**Keyman for iPhone:**
 - Device: iPhone 13
 - OS:  iOS 15.4
 - Keyman version 15.0.231-beta
 - Target application: Keyman

**Comment by mcdurdin** [comment]
So that 'title' is actually associated with the `<` back button... If we wanted to change that, we'd remove that and have a centered title instead?

**Comment by jahorton** [comment]
> So that 'title' is actually associated with the `<` back button... 

Correct.  That's iOS saying "click here to go back to the 'Installed Languages' screen you were on."  Not "this is the 'Installed Languages'" screen.

> If we wanted to change that, we'd remove that and have a centered title instead?

It may be possible to 'customize' the back-button text to just "Back", or something similar.  We could _also_ add a centered title, if we want.
